### PR TITLE
Add COM03-AWS200 - Create an Auto Scaling Group

### PR DESCRIPTION
**Pull request by what-name**

### CODE REVIEW

File: Projects/COM/COM03/COM03-AWS200.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+- What is the difference between "desired capacity" and "minimum capacity"?
+- In which order will EC2 instances be terminated in case of a scale-in event?
+
+## References
+- [What Is Amazon EC2 Auto Scaling?](https://docs.aws.amazon.com/autoscaling/ec2/userguide/what-is-amazon-ec2-auto-scaling.html)
+- [Auto Scaling Groups](https://docs.aws.amazon.com/autoscaling/ec2/userguide/AutoScalingGroup.html)
+- [AWS Auto Scaling FAQs](https://aws.amazon.com/autoscaling/faqs/)
+
+## Tips
+- Use t2.small instances for the Auto Scaling Group to avoid unexpected charges.
```
Comment (omenking) [inline Projects/COM/COM03/COM03-AWS200.md]: t2.micro is free-tier.
t2.small costs something.

**Review by omenking — CHANGES_REQUESTED** [review]
minor change

File: Projects/COM/COM03/COM03-AWS200.md (hunk 1 above)
Comment (what-name) [reply Projects/COM/COM03/COM03-AWS200.md]: ugh, I meant t2.micro... thanks for catching that!

**Review by what-name — COMMENTED** [review]
Changed instance size under Tips section

**Pull request merged**